lib/que/web.rb

```diff
@@ -203,6 +214,11 @@ def search
         "%#{search_param}%"
       end
 
+      def data_search
+        value = params[:value_type] == "int" ? params[:value].to_i : params[:value]
```

Comment:
Wasn't sure how to go about distinguishing values that are integers or strings in the sql query

Reply:
I think rather than pass the value_type in we might do something like:

```ruby
value_type = case(params[:key]) do 
  when 'company_id' 
    :int
  when 'subscription_id'
    :string
  default:
    params[:key]&.index('_id') ? :int : :string
  end
```

Reply:
Hmmm. Not sure how I would use this in a query

Reply:
You wouldn't use this directly in the query, but it's a substitute for requiring the UI to pass it in as a parameter.... The following portion to convert to int would still be required.